new/LIT7128QeneOzyan.xml

```diff
@@ -63,15 +61,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <div type="edition" xml:lang="gez">
                 <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55vc. Division into verses is indicated with ። . 
-                    The end of the poem is indicated with ። ። .</note>
+                    The end of the poem is indicated with ። ። . Verse markers in <ref target="l2"/> and <ref target="l2"/> are not visible because of 
+                    the tight binding.</note>
                 <ab>
                     <l n="1">ዖዝያን፡ ወድቀ፡ እመንበረ፡ ተድላ፡ አመ፡ ፈቀደ፡ ይፀዓን፡ ካህነተ፡ አሮን፡ ሰረገላ።</l> 
                     <l n="2">እንዘ፡ ዲበ፡ ፈረስ፡ ቅይሕ፡ መንግሥተ፡ ገሊላ<supplied reason="undefined" resp="CH">።</supplied></l>
                     <l n="3">ወአመ፡ ፈቀደ፡ በ<del rend="strikethrough"><unclear>ቅ</unclear>ንሣ<unclear>አ፡</unclear></del>ዝንጋዔ፡ ያስተጋብእ፡ 
                         ፍሬ፡ ምህርካሁ፡ ለአማሌቅ፡ ሰግላ።</l>
                     <l n="4">ሳኦል፡ ዘገልገላ።</l>
                     <l n="5">እምላዕለ፡ ርእሱ፡ አውረደ፡ ትርሲተ፡ መንግሥቱ፡ ቀጸላ።</l>   
-                    <l n="6">አባግዕ፡ ጊዜ፡ መብልዕ፡ ከመ፡ ኢይሬእዩ፡ ተኵላ<supplied reason="undefined" resp="CH">።</supplied></l>
+                    <l n="6">አባግዕ፡ ጊዜ፡ መብልዕ፡ ከመ፡ ኢይሬእዩ፡ <supplied reason="undefined" resp="CH">ተኵላ።</supplied></l>
```

Comment:
In fact, on the image with end 23-06 you can see the beginning of the word, if you zoom. It is indeed ተኵላ.